README.rst

```diff
@@ -1,56 +1,114 @@
 
-Django app for linking to help pages with short tokens
+- The application's language might affect which book to display
 
-Overview (please modify)
-------------------------
+This small Django app provides a means to use "help tokens" on the application
+pages, and then use those tokens, and various other settings, to determine the
+actual URL to use.
 
-The ``README.rst`` file should then provide an overview of the code in this
-repository, including the main components and useful entry points for starting
-to understand the code in more detail.
 
 Documentation
 -------------
 
-The full documentation is at https://help-tokens.readthedocs.org.
+Help-tokens provides a context processor, and a redirection URL.  Configuration
+is in a number of settings.
+
+Settings
+........
+
+Help-tokens reads these Django settings to create URLs:
+
+* HELP_TOKENS_INI_FILE: Path to a .ini file containing help token definitions.
+  The format of the ini file is described below.
+
+* HELP_TOKENS_BOOKS: a dictionary mapping book slugs to URLs.  For example::
+
+    HELP_TOKENS_BOOKS = {
+        'learner': 'http://edx.readthedocs.io/projects/learner-guide',
+        'course_author': 'http://edx.readthedocs.io/projects/running-a-course',
+    }
+
+* HELP_TOKENS_VERSION: a string used as part of the final URL, to choose the
+  correct version of the book.  For example, `"latest"`.
+
+* HELP_TOKENS_LANGUAGE_CODE: the language code to use as part of the book URL,
+  mapped through the [locales] section of the ini file.
+
+INI file format
+...............
+
+The .ini file pointed to by HELP_TOKENS_INI_FILE contains the definitions of
+the help tokens themselves.
+
+The `[pages]` section defines the help tokens.  Each help token definition
+consists of a book slug (defined in HELP_TOKENS_BOOKS), a colon, and a URL
+path.  The `default` token is used for missing tokens.  For example::
+
+    [pages]
+    default = learner:index.html
+    instructor = learner:SFD_instructor_dash_help.html
+    course = learner:index.html
+
+    cohortmanual = course_author:course_features/cohorts/cohort_config.html#assign-learners-to-cohorts-manually
+    cohortautomatic = course_author:course_features/cohorts/cohorts_overview.html#all-automated-assignment
+
+The `[locales]` section defines language codes, used with
+HELP_TOKENS_LANGUAGE_CODE to determine the language portion of the URL::
+
+    [locales]
+    default = en
+    en = en
+    en_us = en
+
+
+Context processor
+.................
+
+The context processor is `"help_tokens.context_processor"`.  It adds a function
+`get_online_help_info`.  Call it with a help token, and it will return a dict
+with a `doc_url` entry, the help URL. You can use it like this in a template::
+
+    <a href="${get_online_help_info('visibility')['doc_url']}">...</a>
+
+This interface is a bit verbose, but is to maintain backward compatibility with
+a previous implementation of this context processor.
```

Comment:
I'd be happy to do that in a future commit. Part of the idea of this interface is that there would be other things other than the doc_url to get from the information. There had been a pdf_url, but it was unused, so I removed it from edx-platform.